src/sentry/api/serializers/models/auth_provider.py

```diff
@@ -14,7 +14,7 @@ def serialize(self, obj, attrs, user):
             flags=F("flags").bitand(~OrganizationMember.flags["sso:linked"]),
         ).count()
 
-        login_url = organization.get_url()
+        login_url = Organization.get_url(organization.slug)
```

Comment:
You'll see below, but I started making several functions attached to the Organization instance that really only rely on the slug or id into static methods so that silos could share that logic without needing the ORM object to be shared, too.

Reply:
Should we move to module level functions instead of class methods? It doesn't look like any of the affected methods use the class parameter.